src/pages/Home.vue

```diff
@@ -124,6 +124,16 @@ export default {
           } else {
             this.notifyWarning('error.unknown_happened_header')
           }
+
+          // if existing, redirect to URL path stored before SSO authentication
+          const ssoRedirectUrlPath = window.localStorage.getItem('ssoRedirectUrlPath')
+          window.localStorage.removeItem('ssoRedirectUrlPath')
+
+          const validSsoRedirectUrlPath = ssoRedirectUrlPath &&
+            isString(ssoRedirectUrlPath) &&
+            ssoRedirectUrlPath.startsWith('/')
+
+          if (validSsoRedirectUrlPath) this.$router.push(ssoRedirectUrlPath)
```

Comment:
Redirecting _after_ landing on home page is not ideal user experience.

It would be cleaner to do this at router level with some `before*` hook.